gapis/api/templates/api.go.tmpl

```diff
@@ -415,6 +415,21 @@ import (
     }
   }
 
+func (m {{$name}}) DestroyHandlesʷ(ϟctx context.Context, ϟw ϟapi.StateWatcher) {
+    if m.Map != nil {
+      // Figure this out
```

Comment:
@bjoeris I am not 100% sure what these are supposed to look like.
Any help would be appreciated.